config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -134,3 +134,48 @@ periodics:
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
     testgrid-tab-name: capi-periodic-upgrade-main-1-25-1-26
+
+- name: periodic-cluster-api-provider-azure-e2e-workload-upgrade-1-26-1-27-main
+  interval: 24h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-azure-cred-only: "true"
+    preset-azure-anonymous-pull: "true"
+    preset-azure-capz-sa-cred: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-azure
+    base_ref: main
+    path_alias: sigs.k8s.io/cluster-api-provider-azure
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230703-e6ae5b372a-1.27
+      args:
+        - runner.sh
+        - "./scripts/ci-e2e.sh"
+      env:
+        - name: KUBERNETES_VERSION_UPGRADE_FROM
+          value: "stable-1.26"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "stable-1.27"
+        - name: ETCD_VERSION_UPGRADE_TO
+          value: "3.5.6-0"
+        - name: COREDNS_VERSION_UPGRADE_TO
+          value: "v1.9.3"
```

Comment:
are there any new versions of coredns and etcd? I usually look to the CAPI upgrade tests and copy the values they have

Reply:
Looks like we are using the same versions that CAPI does in their 1.26 -> 1.27 upgrade test:

https://github.com/kubernetes/test-infra/blob/2c129895eabd2b6bb65be76d275f5601f18d13d9/config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml#L243-L253